src/routes/v1/oauth/start/index.unit.test.js

```diff
@@ -0,0 +1,28 @@
+import router from "./index";
+import { providerCfg } from "../../../../lib/oauth/config";
```

Comment:
```suggestion
import { providerCfg } from "oauth/config";
```

@kushalmalani we use a module loader plugin that lets us import under the `src/lib` directory directly (https://github.com/astronomer/houston-api/blob/main/.babelrc#L18).

Reply:
Yeah, that's just VS helping me importing :P. I can update that.